setup.py

```diff
@@ -64,7 +64,6 @@
         "attrs",
         "dynaconf",
         "factory_boy",
-        "oc",
```

Comment:
Is this change related to the removal of yupana?

Reply:
Yes, it is hidden in [camayoc/tests/qpc/yupana/conftest.py](https://github.com/quipucords/camayoc/pull/371/files#diff-de4edbfb273ddf648f427d76d5ac935d2d302c92500f776f701f5883b4847c5e) at line 4.